zebra-chain/src/transaction/sighash.rs

```diff
@@ -782,7 +782,7 @@ mod test {
         let transaction = ZIP243_2.zcash_deserialize_into::<Transaction>()?;
 
         let value = hex::decode("adedf02996510200")?.zcash_deserialize_into::<Amount<_>>()?;
-        let lock_script = Script(hex::decode("00")?);
+        let lock_script = Script::new(vec![]);
```

Comment:
Can we combine the `value` and `lock_script` bytes into a single bytes array, and `zcash_deserialize` them into an `Output`?

That way, we make sure our deserialization and sighash are compatible.

If this works, let's make the same changes for all these unit tests.

Reply:
I'm not sure how helpful would that be... In the test vectors they are listed separately, so I'd have to serialize them by hand to include in the test

Reply:
Ah right. That seems like an issue with the test vectors, but not something we can or should fix.

Our integration tests should catch issues like this anyway.